lib/elevator.js

```diff
@@ -100,7 +106,10 @@ exports.execute = function(command, options, callback) {
  */
 exports.executeSync = function(command, options) {
   try {
-    return child_process.execFileSync(commandBuilder.getBinaryPath(), commandBuilder.build(command, options), {
+    var binaryPath = commandBuilder.getBinaryPath();
+    debug('executing: %s', binaryPath);
```

Comment:
We only use the async version, and this module should go away very soon (we'll switch to native to avoid the dreaded spawn issues), so I guess its fine.